tests/h/cli/commands/user_test.py

```diff
@@ -254,9 +255,16 @@ def password_service(hasher):
 
 
 @pytest.fixture
-def pyramid_config(pyramid_config, signup_service, password_service):
+def delete_user_service(pyramid_request):
+    return DeleteUserService(request=pyramid_request)
```

Comment:
The other tests in this module use real service instances and test the effect on the DB rather than mocking out the service. I've kept the approach here as well.

We could decide not to do that, but we ought to be consistent and probably not add that change into this PR.

Reply:
Yup: that sounds fair.